### ISSUE TITLE
Multi screen support

### ISSUE THREAD

**Issue by Arpit261** [collegeconnect/CollegeConnect#52]
This app does not support multiple screens fix it .....

**Comment by shreelakshmijoshi** [comment]
Hi @Arpit261 !
I am a new open source contributor and would like to solve this issue, could you help me understand the issue better by providing some extra references ? 
Is this is issue open to contribute ?

**Comment by Arpit261** [comment]
Yes ,, I test this app in my phone so some screen are not fit as expected .. So do one thing that you have to test this application and in every aspects of screen 

 As you know testing is more crucial part of every application... 

At the end the app was too good

**Comment by Pranshul456** [comment]
Ok sure i will test the app in multiple screens with different orientations
I will test it with in two days
